.github/workflows/ci.yml

```diff
@@ -63,8 +63,6 @@ jobs:
         platform:
         - ubuntu-latest
         - macos-latest
-        # TODO: Debug the Windows issues
-        # - windows-latest
```

Comment:
I don't see a strong motivation because I would like to know if the CI breaks when the OS is upgraded. On the other hands, we could test a range of versions (probably with Ubuntu), but I'm not very concerned about this.